docs/image-example.rst

```diff
@@ -0,0 +1,253 @@
+Image processing
+----------------
+
+This example will introduce how to use Romanesco to build some simple image
+processing tasks using Pillow_.  We will learn how to chain several tasks
```

Comment:
Somewhat pedantic, but I think we should just call this PIL or Python Imaging Library, since Pillow is just the distribution and that's a detail that users won't really see.